### PR TITLE
Update model awareness

### PR DESCRIPTION
**Pull request by BenoitDherin**
This PR creates a kernel for the uncertainty aware module, protecting it from the libs update. 

It also fixes the kernel removal functionality in two other kernels, where the venv path was wrongly generated:
*  `reinforcement_learning_kernel`
*  `tf_recommenders_kernel`.

### CODE REVIEW

**Comment by takumiohym** [comment]
@BenoitDherin 

These lines can be removed from the main `requirements.txt`?
https://github.com/GoogleCloudPlatform/asl-ml-immersion/blob/331a4181f18edcce9aee8d9636bdd521f09a73a5/requirements.txt#L12-L15

File: notebooks/uncertainty_aware_models/labs/uncertainty_awareness_in_deep_learning.ipynb [hunk 1]
```diff
@@ -98,16 +98,13 @@
    ]
```
Comment (takumiohym) [inline notebooks/uncertainty_aware_models/labs/uncertainty_awareness_in_deep_learning.ipynb:98]: <p>Should add an instruction of how to create this kernel?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

File: notebooks/uncertainty_aware_models/labs/uncertainty_quantification_with_sngp_bert.ipynb [hunk 2]
```diff
@@ -44,14 +44,12 @@
    ]
```
Comment (takumiohym) [inline notebooks/uncertainty_aware_models/labs/uncertainty_quantification_with_sngp_bert.ipynb:44]: <p>Should add an instruction of how to create this kernel?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by takumiohym** [comment]
And this module `README.md` should be updated as well?

https://github.com/GoogleCloudPlatform/asl-ml-immersion/blob/master/notebooks/uncertainty_aware_models/README.md

File: notebooks/uncertainty_aware_models/solutions/uncertainty_quantification_with_sngp_bert.ipynb [hunk 3]
```diff
@@ -43,33 +43,90 @@
     "## Setup"
```
Comment (takumiohym) [inline notebooks/uncertainty_aware_models/solutions/uncertainty_quantification_with_sngp_bert.ipynb:43]: <p>I guess this is remaining here unintentionally.</p><p>(<code>seaborn</code> is included in the requirements.txt)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (takumiohym) [inline notebooks/uncertainty_aware_models/solutions/uncertainty_quantification_with_sngp_bert.ipynb:43]: <p>And this line too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: notebooks/uncertainty_aware_models/labs/uncertainty_awareness_in_deep_learning.ipynb (hunk 1 above)
Comment (BenoitDherin) [reply notebooks/uncertainty_aware_models/labs/uncertainty_awareness_in_deep_learning.ipynb:98]: Done!

File: notebooks/uncertainty_aware_models/labs/uncertainty_quantification_with_sngp_bert.ipynb (hunk 2 above)
Comment (BenoitDherin) [reply notebooks/uncertainty_aware_models/labs/uncertainty_quantification_with_sngp_bert.ipynb:44]: Done!

File: notebooks/uncertainty_aware_models/solutions/uncertainty_quantification_with_sngp_bert.ipynb (hunk 3 above)
Comment (BenoitDherin) [reply notebooks/uncertainty_aware_models/solutions/uncertainty_quantification_with_sngp_bert.ipynb:43]: Nope! Good catch! Removing it.

Comment (BenoitDherin) [reply notebooks/uncertainty_aware_models/solutions/uncertainty_quantification_with_sngp_bert.ipynb:43]: Same. Removing it. Thanks @takumiohym !

**Comment by BenoitDherin** [comment]
@takumiohym Thanks! Incorporated all of your comment and merging now.

**Pull request merged**